pages/mesosphere/dcos/1.13/release-notes/1.13.9/index.md

```diff
@@ -7,6 +7,8 @@ excerpt: Release notes for DC/OS 1.13.9, including Open Source attribution, and
 ---
 DC/OS&trade; 1.13.9 was released on 29 April, 2020.
 
+<p class="message--warning"><strong>WARNING: </strong>The DC/OS 1.13.9 release has an error that will break dcos-net if you upgrade to 2.0.x. We recommend that you do NOT download and upgrade from 1.13.9 to any 2.0.x version at this time. Instead, wait for the 2.0.4 release coming soon.</p>
```

Comment:
Let's not say "has an error"....  Instead:  "The DC/OS 1.13.9 release includes a data format change for the persisted dcos-net state that is not compatible with DC/OS 2.0.0-2.0.3.  If you upgrade from DC/OS 1.13.9 to these versions, dcos-net can crash.   The upcoming DC/OS 2.0.4 release will be compatible with the new data format and is recommended for upgrades from 1.13.9"